client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -27,6 +27,13 @@
 public interface ReaderGroup {
 
     /**
+     * Returns metrics for this reader group.
+     * 
+     * @return a ReaderGroupMetrics object for this reader group.
+     */
+    ReaderGroupMetrics getMetrics();
```

Comment:
Yes, it is my intention that all reader group metrics would go on this interface. As we have at least 3 planned, I figured it made sense to create an interface for them. Also as metrics was an area where we would likely see API changes it makes sense to separate it so that we can both easily call out the non-stability of those method signatures and isolate those changes from the rest of the readerGroup interface.